### PR TITLE
Preparations for 5.0.0.alpha1.snapshot2

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,24 +2,24 @@
...
-gem "logstash-core-event-java", "5.0.0.alpha1.snapshot1"
-gem "logstash-core-plugin-api", "1.4.0"
+gem "logstash-core", "5.0.0.alpha1.snapshot2"
+gem "logstash-core-event-java", "5.0.0.alpha1.snapshot2"
+gem "logstash-core-plugin-api", "1.5.0"
 gem "file-dependencies", "0.1.6"
 gem "ci_reporter_rspec", "1.0.0", :group => :development
-gem "simplecov", :group => :development
-gem "coveralls", :group => :development
+gem "simplecov", ">= 0"
```
Comment (jsvd) [inline Gemfile]: this is strange. these are development gems, why are they're no longer in it?

Comment (ph) [inline Gemfile]: ergh, I ran rake test before I've made the change to the gemfile will revert that.

**Comment by jsvd** [comment]
the lock file also needs to be updated to snapshot2

**Comment by jsvd** [comment]
also 
```
./logstash-core/lib/logstash/version.rb:LOGSTASH_VERSION = "5.0.0-alpha1.snapshot1"
./logstash-core/lib/logstash-core/version.rb:LOGSTASH_CORE_VERSION = "5.0.0-alpha1.snapshot1"
./logstash-core/logstash-core.gemspec:  gem.add_runtime_dependency "logstash-core-event-java", "~> 5.0.0.alpha1.snapshot1"
```

**Comment by ph** [comment]
@jsvd updated

**Comment by jsvd** [comment]
last reference
`./logstash-core-plugin-api/lib/logstash-core-plugin-api/version.rb:# 1.4.0 tracks 5.0.0-alpha1.snapshot1`

**Comment by ph** [comment]
@jsvd my castle for a one command publish.

**Comment by ph** [comment]
Ready?

**Comment by jsvd** [comment]
LGTM

**Pull request closed without merging**